colossalai/kernel/triton/flash_decoding.py

```diff
@@ -9,15 +9,16 @@
 # Triton 2.1.0
 @triton.jit
 def _flash_decoding_fwd_kernel(
-    Q,  # [batch_size, head_num, head_dim]
+    Q,  # [batch_size, head_num, q_len(1), head_dim]
```

Comment:
Is q_len a necessary dimension?

Reply:
Yes - to keep consistency with transformers modeling (e.g. LlamaAttention)